src/CoreArbiterServer.cc

```diff
@@ -1579,6 +1592,8 @@ CoreArbiterServer::installSignalHandler() {
         LOG(ERROR, "Couldn't set signal handler for SIGSEGV");
     if (sigaction(SIGABRT, &signalAction, NULL) != 0)
         LOG(ERROR, "Couldn't set signal handler for SIGABRT");
+    if (sigaction(SIGPIPE, &signalAction, NULL) != 0)
+        LOG(ERROR, "Couldn't set signal handler for SIGPIPE");
```

Comment:
I'm not sure it makes sense to continue the application if you get one of these errors. Let's discuss.